src/walletConnect/v1/reducer.ts

```diff
@@ -7,21 +8,29 @@ import { Actions, UserActions, WalletConnectActions } from 'src/walletConnect/v1
 
 export type PendingAction = { action: WalletConnectPayloadRequest; peerId: string }
 
+export enum WalletConnectDisplayedInfo {
+  None = 'none',
+  Basic = 'basic',
+  Full = 'full',
```

Comment:
Is this used anywhere?

Reply:
it's not yet...i added it to make it look less weird to have a 2 value enum :/ should i remove?